src/main/java/net/dv8tion/jda/api/audio/AudioReceiveHandler.java

```diff
@@ -37,6 +37,20 @@
      *
      * @return If true, JDA enables subsystems to combine all user audio into a single provided data packet.
      */
+    default boolean shouldCombineAudio()
+    {
+        return false;
+    }
+
+    /**
+     * If this method returns true JDA will attempt to provide audio data to this handler by calling
+     * {@link #handleCombinedAudio(CombinedAudio)}. The return value is checked each time JDA attempts receive audio, so if
+     * the developer wanted to start and stop receiving audio it could be done by changing the value returned
+     * by this method at runtime.
```

Comment:
```suggestion
     * If this method returns true JDA will attempt to provide audio data to this handler by calling
     * {@link #handleCombinedAudio(CombinedAudio)}. The return value is checked each time JDA receives audio, so if
     * the developer wanted to start and stop receiving audio it could be done by changing the value returned
     * by this method at runtime.
```